rotkehlchen/tests/conftest.py

```diff
@@ -174,7 +174,7 @@ def get_cassette_dir(request: pytest.FixtureRequest) -> Path:
 
 def is_etherscan_rate_limited(response: dict[str, Any]) -> bool:
     result = False
-    with suppress(JSONDecodeError, KeyError):
+    with suppress(JSONDecodeError, KeyError, UnicodeDecodeError):
```

Comment:
Why this change? What prompted it?

Reply:
For responses that contain bytes, the code under this `suppress` was failing because it was trying to decode a bytes image as a utf-8 string

Reply:
That's not the right solution though.

Look at where this function is called. All responses that go in this function **should be json**.

If this function is called for a bytes response it's an error. I think what you should do is modify the if check in

```python
def vcr_fixture
```

to not go in that function if it's bytes.